projects/RR360/models/losses/a_l1_loss.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Optional
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+import numpy as np
+from mmrotate.registry import MODELS
+from mmdet.models.losses.utils import weighted_loss
+
+@weighted_loss
+def al1_loss(pred: Tensor, target: Tensor) -> Tensor:
```

Comment:
Tell users the difference between the two losses in README.md

Reply:
I Delete the AL1 loss , I find L1 loss achieve high mAP